xwiki-platform-core/xwiki-platform-sandbox/pom.xml

```diff
@@ -35,6 +35,15 @@
     <!-- Name to display by the Extension Manager -->
     <xwiki.extension.name>Sandbox Application</xwiki.extension.name>
   </properties>
+  <dependencies>
+    <dependency>
+      <!-- Needed to display the Sandbox in the Applications panel list -->
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-uiextension-ui</artifactId>
+      <version>${project.version}</version>
+      <type>xar</type>
+    </dependency>
+  </dependencies>
```

Comment:
``xwiki-platform-uiextension-ui`` only has a sheet which I doubt is used by the Sandbox application. The class is generated by ``UIExtensionClassDocumentInitializer`` from the API module.

Reply:
Wrong pick by me indeed, sorry, fixed in latest commit.